.eslintrc.js

```diff
@@ -41,10 +41,6 @@ module.exports = { // eslint-disable-line no-undef
             "error",
             { before: true, after: true },
         ],
-        "linebreak-style": [
-            "error",
-            "unix",
-        ],
```

Comment:
Is it intentional to be able to mix `\n` and `\r\n` in a file? 🤔

Reply:
I have added this back in 6a26342fc762406ac69772804d3b193e490b7e2d. 

I had problems with my local Git configuration because of which I was not able to correctly transform the files in \n only, but this is resolved.